pom.xml

```diff
@@ -86,8 +86,9 @@
             <artifactId>workflow-cps</artifactId>
         </dependency>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>workflow-cps-global-lib</artifactId>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>pipeline-groovy-lib</artifactId>
+						<version>589.vb_a_b_4a_a_8c443c</version> <!-- TODO pending inclusion in BOM -->
```

Comment:
indentation looks a bit deep here

Reply:
it appears this was indented with tabs, yet the other places use spaces.
```suggestion
            <version>589.vb_a_b_4a_a_8c443c</version> <!-- TODO pending inclusion in BOM -->
```

Reply:
Right, sorry…